packages/tabs/src/use-tabs.ts

```diff
@@ -385,6 +389,10 @@ export function useTabPanel(props: Dict) {
   const { isLazy } = useTabsContext()
 
   return {
+    /**
+     * Puts the tabpanel in the page `Tab` sequence.
+     */
+    tabIndex: isSelected ? 0 : -1,
```

Comment:
The main reason we did this is to follow the roving tab-index spec. See this:
https://www.stefanjudis.com/today-i-learned/roving-tabindex/

The roving tab index is a common approach to managing a11y for interactive widgets. Can you test the behavior using only your keyboard? When you do, you might notice that the tabbing goes to the other tabs instead of going into the selected tab panel.

You get what I mean?

Reply:
This makes sense. Given that the tab panel has `hidden` applied to it already, we don't need to add this check.